### PR TITLE
Initial creation

### PR DESCRIPTION
**Pull request by AleksandrsKondratjevs**
There is no graphQl endpoints for product alerts. So create separate module for it. Right now it is small, since can handle only subscribe mutation (same as default magento), but in code it is clearly see that they will implement some kind of managing those subscribes for users and this module with grow more in future.

### CODE REVIEW

File: src/Model/Resolver/ProductAlertSubscribe.php [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+     */
+    public function resolve(
+        Field $field,
+        $context,
+        ResolveInfo $info,
+        array $value = null,
+        array $args = null
+    ) {
+        $productId = (int)$args['productId'];
+        $customerId = (int)$context->getUserId();
```
Comment (zans-laksa) [inline src/Model/Resolver/ProductAlertSubscribe.php]: I see that on the frontend you don't check whether the user is authorized.
But i guess, this feature is for authorized customers only?

Comment (zans-laksa) [reply src/Model/Resolver/ProductAlertSubscribe.php]: This is intended behavior, like in Luma.
The frontend displays the subscribe buttons for any user. But when the unauthorized user clicks the subscribe button, it gives him a notification.

**Review by zans-laksa — APPROVED** [review]
Code review passed.

**Comment by AleksandrsKondratjevs** [comment]
Moved files from src to root.

**Pull request merged**